.eslintrc.js

```diff
@@ -19,7 +19,10 @@ module.exports = {
 
     {
       files: ['*.test.ts', '*.test.js'],
-      extends: ['@metamask/eslint-config-jest'],
+      extends: [
+        '@metamask/eslint-config-jest',
+        '@metamask/eslint-config-nodejs',
```

Comment:
I will update the module template with this change as well. It should be OK to use Node.js imports in tests.